website/identifiers/clients/crossref.py

```diff
@@ -238,8 +238,9 @@ def create_identifier(self, preprint, category, status='public', include_relatio
         else:
             raise NotImplementedError()
 
-    def update_identifier(self, preprint, category):
-        status = self.get_status(preprint)
+    def update_identifier(self, preprint, category, status=None):
+        if status is None:
```

Comment:
Minor: this could be replaced with conditional assignment like `status = status or self.get_status(preprint)`